fc-community-extension/src/birdwatchContentScript.js

```diff
@@ -0,0 +1,21 @@
+import { logger } from "./logging";
+import { alterTwitterNoteSeparator } from "./contentModifiers";
+
+let observer = new MutationObserver(mutations => {
+  for(let mutation of mutations) {
+    for(let addedNode of mutation.addedNodes) {
+      if (addedNode && (typeof addedNode.querySelector) === "function") {
+        const separator = addedNode.querySelector("div.css-175oi2r.r-1p6iasa.r-109y4c4.r-gu4em3");
```

Comment:
Is it a good idea to isolate all the selectors in a file?

Reply:
👍 have fun!